simtools/utils/general.py

```diff
@@ -816,7 +816,7 @@ def read_file_encoded_in_utf_or_latin(file_name):
         with open(file_name, encoding="utf-8") as file:
             lines = file.readlines()
     except UnicodeDecodeError:
-        logging.debug("Unable to decode file using UTF-8. Trying Latin-1.")
+        logging.error("Unable to decode file using UTF-8. Trying Latin-1.")
```

Comment:
Keep this as debug as it is not an `error`. We continue trying to read the file taking the new circumstances into accounts. Errors are raised laterl